src/app/admin/volunteers/shared/form-hours-selector/form-hours-selector.component.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  Component,
+  EventEmitter,
+  Inject,
+  OnDestroy,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { MatFormFieldControl } from '@angular/material/form-field';
+
+@Component({
+  selector: 'app-form-hours-selector',
```

Comment:
> custom component for hours-selection as in figma

Sorry but this component does not look at all how is in figma :|

![image](https://user-images.githubusercontent.com/1881266/103589831-01422c00-4ef5-11eb-95a4-9fdc7fd4e7d0.png)

Reply:
Sorry - according to discussion with Andrey Valcov it was accepted to change such view 
![image](https://user-images.githubusercontent.com/4039092/103620527-505f7f80-4f3c-11eb-9025-97eaec6a7c95.png)
to this one. I will ask for Andrei confirmation and attach it here.

Reply:
so the (figma owner) answer 
Andrei Valcov : @Salut, Este OK asa cum ai facut.@